docs/sensors/smartspot.md

```diff
@@ -1,11 +1,199 @@
-# Creating your own SmartSpot for Air quality
+# User&#39;s guide for Smart Spot Starter Kit
 
-## What you need to order
+## **Introduction**
 
-## How to install and configure it
+In order to offer a simple and intuitive way to get started with our Smart Spot we have developed an Open Source solution called Smart Spot Starter Kit, which has an obvious educational purpose.  Just by simply obtaining this kit and follow this guide you will be able to manage the device and to acquire data from its board sensors, apart from the ones you connect, with your smartphone.
```

Comment:
apply linter, lines should me max 80 chars, see for applied rules:
https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md

and here for the linter detected issues: https://travis-ci.org/smartsdk/guided-tour/jobs/346396767